frontend/lib/src/SessionInfo.ts

```diff
@@ -62,6 +76,16 @@ export class SessionInfo {
     return this._last
   }
 
+  /** Set this._lastForwardMsgID */
+  public setLastForwardMsgID(id: string): void {
+    this._lastForwardMsgID = id
+  }
+
+  /** Return this._lastForwardMsgID */
+  public get lastForwardMsgID(): string | undefined {
+    return this._lastForwardMsgID
+  }
+
```

Comment:
Perhaps I found our SessionInfo class/object a bit wonky, but curious whether we should consider a building a mapping of the session id to this information rather than plug it into the SessionInfo. Not a strong opinion, but something to consider.

Reply:
My guess is that would be overkill + adds some additional layers of complexity for little to no gain. The client only ever cares about one session ID (that is, the previous one it was connected to) when it attempts to reestablish a websocket connection. If it turns out that it can't reconnect to the ID, then it receives a new one and should just drop the last one.